packages/react-instantsearch-core-next/src/__tests__/Index.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { render } from '@testing-library/react';
+// eslint-disable-next-line import/named
+import { IndexWidget } from 'instantsearch.js/es/widgets/index/index';
+import React from 'react';
+
+import { createSearchClient } from '../../../../test/mock';
+import { IndexContext } from '../IndexContext';
+import { InstantSearch } from '../InstantSearch';
+import { Index } from '../SearchIndex';
+
+describe('Index', () => {
+  test('renders children', () => {
+    const searchClient = createSearchClient();
+
+    const { container } = render(
+      <InstantSearch indexName="indexName" searchClient={searchClient}>
+        <Index indexName="childIndex">Children</Index>
```

Comment:
What about `name`? We're already on an `Index` component so it's implied.

Reply:
It's using the same API as InstantSearch Core for alignment.

Reply:
Gotcha. It should be easier to fix this one globally though, we might want to tackle it in the future majors.

Reply:
I don't think we need to shorten it, indexName is a single concept, the name sounds like a label instead of the Algolia concept imo

Reply:
Even if it's a concept, I think there's redundancy in having the term while you're on an `<Index>` widget. We might just bee too used to calling it like that.